core/src/avm2/domain.rs

```diff
@@ -24,6 +24,12 @@ use super::Avm2;
 #[collect(no_drop)]
 pub struct Domain<'gc>(GcCell<'gc, DomainData<'gc>>);
 
+impl std::fmt::Debug for Domain<'_> {
```

Comment:
Accidentally added?

Reply:
My guess is Aaron added it, much like the `Debug` impls later on, while debugging the memory problems caused by not cleaning up loaders.

I'm personally fine with merging the debug impls alongside everything else. Not being able to trace something because someone forgot to add a `Debug` impl is frustrating.